### PR TITLE
Propose Vapor to the SSWG Incubation Process

### PR DESCRIPTION
**Pull request by 0xTim**
Adds a proposal to add Vapor to the SSWG incubation process

### CODE REVIEW

**Comment by 0xTim** [comment]
cc @gwynne

File: proposals/NNNN-vapor.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+[Vapor](https://www.vapor.codes) is a full-featured framework for building server applications and API in Swift. It is a mature framework that has been around since 2016 and has build a large engaged community around it. Vapor provides everything you need to build a server-side Swift application. The main framework provides HTTP routing, middleware, sessions, authentication, websockets, and more. Vapor also provides a suite of packages for working with databases, templating, and other common tasks.
+
+## Motivation
+
+HTTP libraries are a core component of any server-side Swift application. Vapor provides a high-level API for working with HTTP requests and responses to make it easy for developers to build full-featured server applications. It has been around since 2016 and built a large community around it. We believe it should be part of the incubation process to further integrate it into the ecosystem.
+
+## Proposed solution
+
+We propose that the existing MultipartKit implementation be promoted to an officially supported package as is.
```
Comment (adam-fowler) [inline proposals/NNNN-vapor.md]: MultipartKit?

Comment (0xTim) [reply proposals/NNNN-vapor.md]: Copy/paste 🤦

File: proposals/NNNN-vapor.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* Proposal: [SSWG-NNNN](NNNN-vapor.md)
+* Authors: [Tim Condon](https://github.com/0xTim), [Gwynne Raskind](https://github.com/gwynne)
+* Review Manager: TBD
+* Status: **Implemented**
+* Implementation: [vapor/vapor](https://github.com/vapor/vapor)
+* Forum Threads: [Proposal - TBD](), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
+
+## Package Description
+
+[Vapor](https://www.vapor.codes) is a framework for building server applications, APIS and backends in Swift.
```
Comment (adam-fowler) [inline proposals/NNNN-vapor.md]:
```suggestion
[Vapor](https://www.vapor.codes) is a framework for building server applications, APIs and backends in Swift.
```

File: proposals/NNNN-vapor.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Vapor
+
+* Proposal: [SSWG-NNNN](NNNN-vapor.md)
+* Authors: [Tim Condon](https://github.com/0xTim), [Gwynne Raskind](https://github.com/gwynne)
+* Review Manager: TBD
+* Status: **Implemented**
+* Implementation: [vapor/vapor](https://github.com/vapor/vapor)
+* Forum Threads: [Proposal - TBD](), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
```
Comment (adam-fowler) [inline proposals/NNNN-vapor.md]:
```suggestion
* Forum Threads: [Pitch](https://forums.swift.org/t/pitch-vapor/70660)
```

File: proposals/NNNN-vapor.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# Vapor
+
+* Proposal: [SSWG-NNNN](NNNN-vapor.md)
+* Authors: [Tim Condon](https://github.com/0xTim), [Gwynne Raskind](https://github.com/gwynne)
+* Review Manager: TBD
```
Comment (adam-fowler) [inline proposals/NNNN-vapor.md]:
```suggestion
* Review Manager: Adam Fowler
```

**Review by adam-fowler — APPROVED** [review]
LGTM, some minor updates suggested

**Pull request merged**